.github/workflows/wheel_linux_aarch64.yml

```diff
@@ -0,0 +1,206 @@
+        container_img: ["quay.io/pypa/manylinux2014_aarch64"]
+
+    name: ${{ matrix.os }}::${{ matrix.arch }} - ${{ matrix.pl_backend }} (Python ${{ fromJson('{ "cp38-*":"3.8","cp39-*":"3.9","cp310-*":"3.10","cp311-*":"3.11" }')[matrix.cibw_build] }})
+    runs-on: ${{ matrix.os }}
+
+    steps:
+      - name: Cancel Previous Runs
+        uses: styfle/cancel-workflow-action@0.10.0
+        with:
+          access_token: ${{ github.token }}
+
+      - uses: actions/checkout@v3
+
+      - name: Restoring cached dependencies
+        id: kokkos-cache
+        uses: actions/cache@v3
+        with:
+          path: ${{ github.workspace }}/Kokkos_install/${{ matrix.exec_model }}
+          key: ${{ matrix.container_img }}-kokkos${{ matrix.kokkos_version }}-${{ matrix.exec_model }}
+
+      - name: Copy cached libraries
+        run: |
+          mkdir Kokkos
+          cp -rf ${{ github.workspace }}/Kokkos_install/${{ matrix.exec_model }}/* Kokkos/
+
+      - name: Install cibuildwheel
+        run: python3 -m pip install cibuildwheel~=2.11.0
+
+      - uses: docker/setup-qemu-action@v2
+        name: Set up QEMU
+
+      - name: Build wheels
+        env:
+          CIBW_ARCHS_LINUX: ${{matrix.arch}}
+
+          CIBW_BUILD: ${{ matrix.cibw_build }}
+
+          CIBW_SKIP: "*-musllinux*"
+
+          # Python build settings
+          CIBW_BEFORE_BUILD: |
+            cat /etc/yum.conf | sed "s/\[main\]/\[main\]\ntimeout=5/g" > /etc/yum.conf
+            pip install ninja cmake~=3.24.0
+
+          CIBW_MANYLINUX_AARCH64_IMAGE: manylinux2014
+
+          CIBW_BUILD_VERBOSITY: 3
+
+        run: python3 -m cibuildwheel --output-dir wheelhouse
+
+      - name: Validate wheels
+        run: |
+         python3 -m pip install twine
+         python3 -m twine check ./wheelhouse/*.whl
+
+      - uses: actions-ecosystem/action-regex-match@v2
+        id: rc_build
+        with:
+          text: ${{ github.event.pull_request.head.ref }}
+          regex: '.*[0-9]+.[0-9]+.[0-9]+[-_]?rc[0-9]+'
+
+      - uses: actions/upload-artifact@v2
+        # uncomment before merging PR
+        # if: ${{ github.event_name == 'release' || github.ref == 'refs/heads/master' }}
+        with:
+          name: ${{ runner.os }}-wheels-${{ matrix.pl_backend }}-${{ matrix.arch }}.zip
+          path: ./wheelhouse/*.whl
+
+  # Refine and uncomment before merging to pennylane-lightning repository
+  # upload-pypi:
+  #   needs: linux-wheels-aarch64
+  #   runs-on: ubuntu-latest
+  #   if: ${{ github.event_name == 'release' || github.ref == 'refs/heads/master'}}
+  #   steps:
+  #     - uses: actions/download-artifact@v2
+  #       with:
+  #         name: Linux-wheels-aarch64.zip
+  #         path: dist
+
+  #     - name: Upload wheels to PyPI
+  #       uses: pypa/gh-action-pypi-publish@release/v1
+  #       with:
+  #         user: __token__
+  #         password: ${{ secrets.TEST_PYPI_API_TOKEN }}
+  #         repository_url: https://test.pypi.org/legacy/
```

Comment:
Should we keep this for now?

Reply:
I'm not familiar with PyPI usage, but I assumed we don't have what is needed in place to upload wheels from this repository.

Reply:
Sorry, do you mean the commented code? If that's what you are referring to, I keep it here because this will be immediately used when opening the PR on Lightning.